test/controllers/comments_controller_test.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class CommentsControllerTest < ActionDispatch::IntegrationTest
+  test "コメントの作成に成功した場合、法案詳細ページにリダイレクトされる" do
+    bill = bills(:one)
+    assert_difference "bill.comments.count" do
+      post bill_comments_url(bill), params: { comment: { description: "test" } }
+    end
+    assert_redirected_to bill_url(bill)
+  end
+
+  test "コメントの作成に失敗した（空のコメントを投稿した場合）場合、法案詳細ページにリダイレクトされる" do
+    bill = bills(:one)
+    assert_no_difference "bill.comments.count" do
```

Comment:
[MUST] test/controllers/bills_controller_test.rb#L8 と同様

```suggestion
    assert_no_difference -> { bill.comments.count" } do
```